src/OpenLoco/src/Vehicles/RoutingManager.cpp

```diff
@@ -51,6 +52,12 @@ namespace OpenLoco::Vehicles::RoutingManager
         std::fill(std::begin(vehRoutingArr), std::end(vehRoutingArr), kRoutingNull);
     }
 
+    // 0x004A8810
+    void resetRoutingTable()
+    {
+        std::fill_n(&routings()[0][0], Limits::kMaxVehicles * Limits::kMaxRoutingsPerVehicle, kRoutingNull);
```

Comment:
Can't you just use begin and end on this?

Reply:
I think that's not really compatible with how the array is defined. Perhaps I'm missing something, but this doesn't work, anyway:
```cpp
        std::fill(std::begin(routings()), std::end(routings()), kRoutingNull);
```